src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -468,11 +468,16 @@ internal MirrorData GetCachedValueFromEngine(EngineController engine)
         }
 
         /// <summary>
-        /// Use to indicated if a node was involved in the most recent graph evaluation.
+        /// Use to indicate if a node was involved in the most recent graph evaluation.
```

Comment:
Let's rephrase this one to make the design intent clear for future references:

```csharp
/// This flag is used to determine if a node was involved in a recent execution.
/// The primary purpose of this flag is to determine if the node's render packages 
/// should be returned to client browser when it requests for them. This is mainly 
/// to avoid returning redundant data that has not changed during an execution.
```